apps/glusterfs/brick_entry.go

```diff
@@ -213,6 +214,7 @@ func (b *BrickEntry) Destroy(db wdb.RODB, executor executors.Executor) error {
 	req.Size = b.Info.Size
 	req.TpSize = b.TpSize
 	req.VgId = b.Info.DeviceId
+	req.Path = strings.TrimSuffix(b.Info.Path, "/brick")
```

Comment:
There are a bunch of brick & path helper functions in pkg/utils/paths.go. Would any of them be appropriate for this?